mapfishapp - actions - jsbuild not found

https://github.com/georchestra/georchestra/commit/e79986dd52353953f6677418679c8be497afe652/checks?check_suite_id=437192082
```
[INFO] --- exec-maven-plugin:1.5.0:exec (default) @ mapfishapp ---
/home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: 29: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/env/bin/jsbuild: not found
creating virtual env and installing jstools...
/home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: 33: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: /usr/bin/virtualenv: not found
/home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: 34: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/env/bin/pip: not found
done.
running jsbuild for main app...
/home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: 39: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/build.sh: /home/runner/work/georchestra/georchestra/mapfishapp/jsbuild/env/bin/jsbuild: not found
done.
[NOK] jsbuild failure
[ERROR] Command execution failed.
```

--- a/.github/workflows/mapfishapp.yml
+++ b/.github/workflows/mapfishapp.yml
@@ -16,6 +16,10 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: '8.x'
+    - name: setup Python 2.x (for jsbuild)
+      uses: actions/setup-python@v1
+      with:
+        python-version: '2.x'
     - name: "Configuring Maven"
       run: |
           mkdir -p $HOME/.m2

